src/components/topic/TopicService.js

```diff
@@ -71,10 +71,15 @@ goog.require('ga_permalink');
         });
 
         // Returns a promise that is resolved when topics are loaded
-        this.getTopics = function() {
+        this.loadConfig = function() {
           return topicsP;
```

Comment:
If components are aware of it, then ok.

But there's another race condition :=)

If a component uses `gaTopic.getTopics` in their 'then' function before the service's 'then' function is called, it will be undefined, no? (same for gaTopic.get())